src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double randomRadius;

```

Comment:
Let's rename it to simply `radius`. It does not matter what will be set - random or actual radius.